docs/discovery.md

```diff
@@ -24,6 +24,11 @@ can use.
 
 ### Hosted Discovery with Docker Hub
 
+#####The Hosted Discovery Service is not recomended for production use.
+#####It's intended to be used for testing/development.
```

Comment:
Maybe we can add a sentence since testing may DoS the site. `Please do not send high volume requests to this discovery service.`